contributors/triage/README.md

```diff
@@ -0,0 +1,13 @@
+# Kubernetes Issue Triage
```

Comment:
@bgrant0607 ideally this should be for all repos i.e.  kubernetes/kubernetes and related kubernetes repos. So how about I add a line in the content to make it clear? I guess we can leave title as such. Thanks!